Quantum-Control-Applications/Superconducting/Single-Fixed-Transmon/configuration_with_mw_fem.py

```diff
@@ -0,0 +1,397 @@
+"""
+QUA-Config supporting OPX1000 w/ MW-FEM
+"""
+
+from pathlib import Path
+import numpy as np
+from qualang_tools.config.waveform_tools import drag_gaussian_pulse_waveforms
+from qualang_tools.units import unit
+
+######################
+# Network parameters #
+######################
+qop_ip = "127.0.0.1"  # Write the QM router IP address
+cluster_name = None  # Write your cluster_name if version >= QOP220
+qop_port = None  # Write the QOP port if version < QOP220
+
+octave_config = None
+
+con = "con1"
+fem = 1
```

Comment:
This might sound "stupid", but, currently in our SaaS simulator, FEMs 1-4 are LF and 5-8 are MW.
In addition, it seems that the error message you get, if you try to use MW-FEM in slot 1, is not good.
So I suggest we change (in all files) the default MW-FEM to be 5.
@deanpoulos @TheoLaudatQM

Reply:
Will do